### DISCUSSION TITLE
Change Tracking Binary Edits?

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#280] [General]
Is there a right way to change track the edits to things like maps, shape metadata, etc? Ones that are not simple line changes that would appear in a diff but ones that are stored and exchanged in compiled/binary formats?

For example, I can add a shape with a SHP file, but need to add the name, offsets, values, etc.

**Comment by DominusExult** [comment]
I can't think of anything but excessive git usage (local perhaps)
